examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>We could maybe add a quick note about making sure that the mlp blocks used for the user and query towers should have the same last dimension. This is needed because we will compute the dot product between the two towers' outputs to get the similarity scores. </p><p>We can also set a param <code>tower_dim=64</code> and pass it the MLPBlocks:  <code>mm.MLPBlock([128, tower_dim])</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>